daphne/src/roles_test.rs

```diff
@@ -704,7 +653,7 @@ async fn http_post_collect_fail_invalid_batch_interval() {
     let leader = MockAggregator::new();
     let task_id = leader.nominal_task_id();
     let task_config = leader.get_task_config_for(task_id).unwrap();
-    let now = leader.get_current_time();
```

Comment:
You could continue to use `get_current_time()`, but just have it always return the same value.

Reply:
We are using the `now` that is initialized when creating a new `MockAggregator` via `MockAggregator.now`